noir_stdlib/src/ec/tecurve.nr

```diff
@@ -354,11 +354,9 @@ mod curvegroup {
 
         // Scalar multiplication (p + ... + p n times)
         pub fn mul(self, n: Field, p: Point) -> Point {
-            let N_BITS = crate::field::modulus_num_bits();
```

Comment:
Just taking note this is another place we are ingraining the modulus size

Reply:
Note that we're harcoding 254 already anyway on the next line so :shrug:. We need to get nargo building with a different field to tackle this properly.

Reply:
Didn't notice that next line (although I do think it can be removed based on the comment). Just made the comment as while going through the PR I noticed these extra places we were assuming 254 I didn't realize we were.